owncloudApp/src/main/res/layout/edit_box_dialog.xml

```diff
@@ -17,8 +17,9 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<LinearLayout
+<com.google.android.material.textfield.TextInputLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/user_input_text_layout"
```

Comment:
Maybe it's better if we name the ID differently, since this is a very general name and just by seeing it we cannot infer that it belongs to this dialog, or even worse, confuse it in a future if we add some more TextInputLayout and we retrieve it with `findViewById` 😓. A proposal:
```suggestion
    android:id="@+id/edit_box_input_text_layout"
```

Reply:
Done